axonserver/src/main/java/io/axoniq/axonserver/message/query/QueryDispatcher.java

```diff
@@ -234,10 +234,10 @@ public void dispatchProxied(SerializedQuery serializedQuery, Consumer<QueryRespo
             try {
                 queryCache.put(key, queryInformation);
                 dispatchOne(queryHandler, serializedQuery, timeout);
-            } catch (InsufficientCacheCapacityException insufficientCacheCapacityException) {
+            } catch (InsufficientBufferCapacityException insufficientBufferCapacityException) {
                 queryInformation.completeWithError(queryHandler.getClientId(),
                                                    ErrorCode.QUERY_DISPATCH_ERROR,
```

Comment:
Use ErrorCode.TOO_MANY_REQUESTS here

Reply:
its there if you open file to view changes, but for some reason change is not shown here